docs/api/api_blueprint.apib

```diff
@@ -4423,6 +4422,323 @@ Delete a single notification.
           }
         }
 
+# Group Discount Codes
+Discount Codes related to the events.
+
+| Parameter | Description | Type | Required |
+|:----------|-------------|------|----------|
+| `code`  | Discount Code | string | - |
+| `discount-url` | URL of the discount | string | - |
+| `value` | Value of the discount | float | - |
+| `type` | Amount or Percentage | string | - |
+| `is-active` | If the discount code is active | boolean | - |
+| `tickets-number` | Tickets Number for Discount Code | integer (>=0) | - |
+| `min-quantity` | Minimum number of tickets for Discount Code | integer (>=0) | - |
+| `max-quantity` | Maximum number of tickets for Discount Code | integer (>=0) | - |
+| `valid-from` | Discount Code valid from | ISO 8601 (tz-aware) | - |
+| `used-for` | Discount Code used for | String | - |
+
```

Comment:
@enigmaeth description for `tickets`, `created-at` and `valid-till` fields are missing